machine/wnc_x86_64/rootconf/sysroot-lib-onie/sysinfo-platform

```diff
@@ -0,0 +1,48 @@
+get_serial_num()
+{
+    if [ -z "$serial_num" ] ; then
+        if type get_serial_num_platform | grep -q 'shell function' > /dev/null 2>&1 ; then
+            serial_num=$(get_serial_num_platform)
+        elif type get_serial_num_arch | grep -q 'shell function' > /dev/null 2>&1 ; then
+                serial_num=$(get_serial_num_arch)
+        fi
+    fi
+
+    if [ -n "$serial_num" ] ; then
+        echo $serial_num
+    else
+        echo "Unknown"
+    fi
+}
+
+# Use the contents of /sys/class/net/eth0/address
+
+get_ethaddr()
+{
+   if [ -n "$onie_eth_addr" ] ; then
+       eth_addr="$onie_eth_addr"
+   fi
+   if [ -z "$eth_addr" ] ; then
+        if type get_ethaddr_platform | grep -q 'shell function' > /dev/null 2>&1 ; then
+            eth_addr=$(get_ethaddr_platform)
+        elif type get_ethaddr_arch | grep -q 'shell function' > /dev/null 2>&1 ; then
+            eth_addr=$(get_ethaddr_arch)
+        elif [ -r /sys/class/net/eth0/address ] ; then
+            # Print Ethernet address from eth0 interface
+            eth_addr=$(cat /sys/class/net/eth0/address)
+        fi
+    fi
+
+    if [ -n "$eth_addr" ] ; then
+        echo $eth_addr
+    else
+        echo "Unknown"
+    fi
+
+#    [ -r /sys/class/net/eth0/address ] && cat /sys/class/net/eth0/address
+}
```

Comment:
Is this function different from the default?  If not then this function definition is not required.